exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `greeting`
 
+let greeting = 'Hello World';
```

Comment:
`let` works fine here, but if the variable is only going to be set once, `const` is better